frontend/src/constants.js

```diff
@@ -41,44 +41,16 @@ export const featureColors = JSON.parse(`{
     "entertaining_relaxing": "#6ba4ff"
 }`);
 
-// Dictionary mapping video report fields to their descriptions
-export const videoReportFieldNames = JSON.parse(`{
-    "troll": "The content is a troll",
-    "kid": "The content is not kid-friendly",
-    "misl0": "The content is slightly misleading",
-    "misl1": "The content is very misleading",
-    "misl2": "The misinformation of the content is misleading and dangerous",
-    "hate": "The content contains hate speech",
-    "violence": "The content promotes violence",
-    "bully": "The content is cyber-bullying",
-    "threat": "The content contains threats",
-    "violent_act": "The content calls for violent acts",
-    "death_threat": "The content contains death threats"
-}`);
-
```

Comment:
The constants.js file is auto-generated from constants.py, so the change needs to be done there

Reply:
I removed my changes to this file